test/Microsoft.AspNet.Mvc.FunctionalTests/TagHelpersTest.cs

```diff
@@ -54,38 +54,62 @@ public class TagHelpersTests
             Assert.Equal(expectedContent, responseContent);
         }
 
-        public static IEnumerable<object[]> TagHelpersAreInheritedFromViewStartPagesData
+        public static TheoryData TagHelpersAreInheritedFromGlobalImportPagesData
         {
             get
             {
-                var expected1 =
-@"<root>root-content</root>
-
-
-<nested>nested-content</nested>";
-                yield return new[] { "NestedViewStartTagHelper", expected1 };
-
-                var expected2 =
-@"layout:<root>root-content</root>
-
-
-<nested>nested-content</nested>";
-
-                yield return new[] { "ViewWithLayoutAndNestedTagHelper", expected2 };
-
-                var expected3 =
-@"layout:<root>root-content</root>
-
-
-page:<root/>
-<nested>nested-content</nested>";
-                yield return new[] { "ViewWithInheritedRemoveTagHelper", expected3 };
+                // action, expected
+                return new TheoryData<string, string>
+                {
+                    {
+                        "NestedGlobalImportTagHelper",
+                        string.Format(
+                            "<root>root-content</root>{0}{0}{0}<nested>nested-content</nested>",
+                            Environment.NewLine)
```

Comment:
if the view uses HTML or tag helpers, the response HTML will likely contain generated newlines.  so the real fix here is for the functional tests to behave like browsers and normalize the returned line endings before comparing.  in the meantime, there shouldn't be a hard-and-fast rule about those comparisons.